.travis.yml

```diff
@@ -12,7 +12,7 @@ env:
   global:
     - PATH=$HOME/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/bin:$HOME/.cargo/bin:$PATH
     # Habitat Rust program components
-    - _RUST_HAB_BIN_COMPONENTS="components/airlock"
+    - _RUST_HAB_BIN_COMPONENTS=""
```

Comment:
Is this worth keeping at all if it's empty?

Reply:
Yeah I started to remove all the bin plumbing  but we may have a new bin tool in the (near to mid term) future so leaving it in for now..